src/widgets/ActivityDetails/model/mappers.ts

```diff
@@ -48,11 +49,29 @@ function convertToTextAnswer(item: TextItem): { answer: TextAnswerPayload | null
   }
 }
 
-function convertToSingleSelectAnswer(item: RadioItem): { answer: SingleSelectAnswerPayload | null; itemId: string } {
+function convertToSingleSelectAnswer(item: RadioItem): {
+  answer: SingleSelectAnswerPayload | null
+  itemId: string
+  alert: Array<AlertDTO>
```

Comment:
Alerts should be mapped in a different function

Reply:
Done